build/yamls/base/conf/antrea-agent.conf

```diff
@@ -31,8 +31,9 @@ featureGates:
 # Enable controlling SNAT IPs of Pod egress traffic.
 #  Egress: false
 
-# Enable flexible IPAM mode for Antrea. This mode allows to assign IP Ranges to Namespaces,
-# Deployments and StatefulSets via IP Pool annotation.
+# Enable AntreaIPAM. AntreaIPAM is required by the bridging mode, and can assign IPs to bridging
+# mode Pods from the IPPools configured for Namespaces, Deployments, StatefulSets, or individual
+# Pods.
```

Comment:
I don't know if this is the right place to mention it but as it is phrased at the moment, it makes it sound like `AntreaIPAM` only make sense in bridging mode. While it may be the case now, I believe that the intention of this PR is to support other use cases for `AntreaIPAM`, which won't require bridging mode.

Reply:
Sure. I can remove "from the IPPools configured for Namespaces, Deployments, StatefulSets, or individual Pods".